tests/test_odcs_util.py

```diff
@@ -82,13 +82,16 @@ def compose_json(state, state_name, source_type='module', source=MODULE_NSV,
     ['x86_64'],
     ['breakfast', 'lunch'],
 ))
-@pytest.mark.parametrize(('source', 'source_type', 'packages', 'expected_packages', 'sigkeys'), (
-    (MODULE_NSV, 'module', None, None, None),
-    ('my-tag', 'tag', None, [], None),
-    ('my-tag', 'tag', ['spam', 'bacon', 'eggs'], ['spam', 'bacon', 'eggs'], None),
-    ('my-tag', 'tag', ['spam', 'bacon', 'eggs'], ['spam', 'bacon', 'eggs'], ['B456', 'R123']),
-    ('my-tag', 'tag', ['spam', 'bacon', 'eggs'], ['spam', 'bacon', 'eggs'], ""),
-    ('my-tag', 'tag', ['spam', 'bacon', 'eggs'], ['spam', 'bacon', 'eggs'], []),
+@pytest.mark.parametrize(('source', 'source_type', 'packages', 'expected_packages', 'sigkeys',
+                          'modular_koji_tags', 'expected_koji_tags'), (
+    (MODULE_NSV, 'module', None, None, None, [], None),
+    (MODULE_NSV, 'module', None, None, None, ['release'], ['release']),
+    ('my-tag', 'tag', None, [], None, 'release', None),
```

Comment:
Should that be `['release']`?

Reply:
No, modular koji tags are not a valid parameter for 'tag' releases and get ignored.